include/deal.II/fe/mapping.h

```diff
@@ -336,6 +336,23 @@ class Mapping : public Subscriptor
   get_vertices(
     const typename Triangulation<dim, spacedim>::cell_iterator &cell) const;
 
+  /**
+   * Return the mapped center of a cell.
+   *
+   * Most of the time, these will simply be the center of a cell as returned by
+   * <code>cell-@>center()</code>. However, there are also mappings that add
```

Comment:
I think to understand what you mean by "most of the time", you probably want to define what exactly you call the "center". We do have `cell->barycenter()` already.

I also wouldn't say "Most of the time", but specify under what conditions this would be. Namely, "If you are using a (bi-,tri-)linear mapping, this function simply returns the value also produced by `cell->center()`."